desktop/windows/release-notes/index.md

```diff
@@ -23,15 +23,48 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.6.1
-2022-03-22
+## Docker Desktop 4.7.0
+2022-04-07
 
 > Download Docker Desktop
 >
 > [For
 > Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
 > .button .primary-btn }
 
+### Security
+
+- Update Docker Engine to v20.10.14 to address [CVE-2022-24769](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24769)
+- Update containerd to v1.5.11 to address [CVE-2022-24769](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24769)
+
+### New
+
+- Add installer flags and [instructions](https://docs.docker.com/desktop/windows/install/#install-from-the-command-line) for IT to install Docker Desktop remotely
+- Add [Docker SBOM plugin v0.6.0](https://github.com/docker/sbom-cli-plugin/releases/tag/v0.6.0)
+
+### Upgrades
+
+- [Docker Engine v20.10.14](https://docs.docker.com/engine/release-notes/#201014)
+- [Compose v2.4.1](https://github.com/docker/compose/releases/tag/v2.4.1)
+- [Buildx 0.8.2](https://github.com/docker/buildx/releases/tag/v0.8.2)
+- [containerd v1.5.11](https://github.com/containerd/containerd/releases/tag/v1.5.11)
+- [Go 1.18](https://golang.org/doc/go1.18)
+
+### Bug fixes and minor changes
+ - Use `cri-dockerd` version 0.2.0 for new Kubernetes clusters.
+ - Fixed a bug where the Registry Access Management policy would never be download if the first attempt to refresh fails.
+ - Fixed startup hang when osxfs is used for file sharing.
+ - Fixed volume title. Fixes [docker/for-win#12616](https://github.com/docker/for-win/issues/12616).
+ - Improved login feedback.
+ - Fixed a bug in WSL2 distro integration causing docker command to stop working from distros after a restart or a switch to windows containers.
```

Comment:
I'm finding it hard to parse this sentence.

Reply:
How about "Fixed a bug in the WSL 2 integration that caused Docker commands to stop working after restarting Docker Desktop or after switching to Windows containers."

Reply:
Not sure if there's specific command that didnt work, or if it's all Docker commands?

Reply:
Not specific ones, all commands failed because proxy was stuck